bin.src/comcamCloseLoop.py

```diff
@@ -326,6 +326,13 @@ def _useMinDofIdx(ofcCalc):
 
     ztaac.setZkAndDofInGroups(m1m3Bend=m1m3Bend, m2Bend=m2Bend)
 
+def eraseFolderContent(targetDir):
```

Comment:
It is recommended in Python to begin the function name with "_" to imply the internal use. Therefore, use `_eraseFolderContent()` might be better in this scenario. Another choice is to put this function to Utility.py if you foresee there will be multiple uses of this function as a public function, which means you do not add "_" in the function name. If this is the case, you need to add the doc string with `numpy` format to follow DM's developer manual and the unit test as well.